playbooks/roles/hammer-devel/tasks/hammer_cli_katello.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: 'Clone the hammer-cli-katello repository'
+  git: repo=https://github.com/katello/hammer-cli-katello.git dest=~/hammer-cli-katello update=no
+- name: 'Add local gems to Gemfile.local'
+  blockinfile:
+    dest: ~/hammer-cli-katello/Gemfile.local
+    create: yes
+    block: |
+      gem 'hammer_cli', :path => '../hammer-cli'
+      gem 'hammer_cli_foreman', :path => '../hammer-cli-foreman'
+      gem 'hammer_cli_foreman_tasks', :path => '../hammer-cli-foreman-tasks'
+- name: 'Remove hammer_cli_foreman in Gemfile'
+  lineinfile:
+    dest: ~/hammer-cli-katello/Gemfile
+    regexp: "gem 'hammer_cli_foreman'"
+    line: ''
+- name: 'Remove hammer_cli in Gemfile'
+  lineinfile:
+    dest: ~/hammer-cli-katello/Gemfile
+    regexp: "gem 'hammer_cli'"
+    line: ''
+- name: 'Install bundler'
+  command: bash -lc 'gem install bundler'
```

Comment:
Could you use the gem module here? http://docs.ansible.com/ansible/gem_module.html